engine/src/flutter/lib/web_ui/test/engine/composition_test.dart

```diff
@@ -147,12 +137,14 @@ Future<void> testMain() async {
       });
 
       test('should return editing state if extentOffset is smaller than composingText length', () {
-        const String composingText = 'composeMe';
+        const String composingText = '今日は';
 
         final EditingState editingState = EditingState(
-          text: 'Test',
+          text: '今日は',
           baseOffset: 0,
-          extentOffset: 4,
+          extentOffset: 2,
+          composingBaseOffset: 0,
```

Comment:
I think the composing range here should actually be `(0,3)` since it should cover the full length of the `composingText`.